_posts/2022-10-28-multiple-data-source.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title:  "Launch Highlight: Multi Data Source"
+authors:
+  - tyarong
+  - zengyan
+  - kroosh
+date:   2022-10-28 00:00:00 -0700
+categories:
+  - community
+---
+
+The current architecture of OpenSearch Dashboards limits it to stick with one single OpenSearch cluster. When users need to see their data in different OpenSearch clusters, they need to have a complete OpenSearch with Dashboards stack setup, and navigate between different OpenSearch Dashboards endpoints.
+
+To solve these challenges, we are excited to announce Multi Data Source — the new experimental feature to support multiple data source connections in OpenSearch Dashboards.
+
+
+###Benefits
+
+* **Cost Saving**: For large enterprise customer which has hundreds of OpenSearch/Elasticsearch cluster, they don’t need to create and maintain same amount of OpenSearch Dashboards server.
+* **Performance**: OpenSearch Dashboards health check will cause CPU, Memory, Network overhead to OpenSearch cluster. In shared node architecture (OpenSearch Dashboards and OpenSearch running on same node), it may cause 10% CPU and System Memory usage.
+* **Usability**:
+  * As someone who is responsible for managing employee access to sensitive data, no longer need to manage credentials across several clusters and configure, install, provision, admin multiple OpenSearch Dashboards.
+  * As someone who builds reports across multiple teams, no longer need to navigate across multiple OpenSearch Dashboards applications in order to see all data needed, or manually build reports combining the data from different sources.
+* **Flexibility**: Customer could add and connect to any compatible data sources, and remove them anytime.
+
+###What can you do with multi data source experimental launch?
+
+The first phase of multiple data source will provide OpenSearch Dashboards ability to query data from multiple OpenSearch endpoints with compatible versions, with this feature you can:
+
+* Add/remove the data sources which is compatible with Elasticsearch DSL, e.g. AWS OpenSearch Domains, AWS OpenSearch Serverless collections, and customer self-managed OpenSearch clusters
+* Create visualizations that renders data from different index patterns to compare time-series data in the same dashboard
```

Comment:
Suggested revision:

Create visualizations comparing time-series data from different index patterns and combine that data in a single dashboard.